### ISSUE TITLE
Convert standard single and double quotes to open / close quotes

### ISSUE THREAD

**Issue by jhung** [fluid-project/floeproject.org#243]
## Is your feature request related to a problem?
In much of the content, quotes are straight quotes which can be hard to parse.

## Describe the solution you'd like
Convert all single and double quotes to open and close (aka curling left and right) quotes.

## Additional context or notes
See this page for the HTML codes for curling open and close quotes:
https://www.w3schools.com/CSSref/pr_gen_quotes.asp

**Comment by jobara** [comment]
This can be mostly done by adding the following to the .eleventy.js file.

```JavaScript
    // Configure markdown to use smartquotes
    let markdownIt = require("markdown-it");
    let options = {
        html: true,
        typographer: true
    };
    config.setLibrary("md", markdownIt(options));
```

This makes uses of the [typegrapher setting from markdown-it](https://github.com/markdown-it/markdown-it#init-with-presets-and-options). 

However, things that aren't processed by markdown won't be updated. Most notably this is in the excerpts used for the news articles. 

When 11ty 1.0 is released it will come with a [Render plugin](https://www.11ty.dev/docs/plugins/render/), that will allow us to add a shortcode for wrapping the excerpt in, so that it is processed by Markdown in the nunchucks template. 

[eleventy-plugin-fluid](https://github.com/fluid-project/eleventy-plugin-fluid) does come with a markdown shortcode, but unfortunately there isn't a way to configure it. see https://github.com/fluid-project/eleventy-plugin-fluid/issues/85
